### PR TITLE
Added "Gore Longitude", fiction book inspired by "Blood Meridian"

### PR DESCRIPTION
**Pull request by SpasmodicReveler**
#### Summary
None

#### Purpose of change
Adding a fiction book titled "Gore Longitude".

#### Describe the solution
I added the relevant .json to the misc.json file in the books section.

#### Testing
I started a test save and debugged the book into the game.

### CODE REVIEW

File: data/json/items/book/misc.json [hunk 1]
```diff
@@ -347,6 +347,17 @@
     "time": "20 m",
     "fun": 3
   },
+  {
+    "id": "book_fict_soft_west_gore",
+    "type": "BOOK",
+    "name": { "str": "Gore Longtitude", "str_pl": "copies of Gore Longitude" },
+    "description": "Award winning author Cornac McCaffordy weaves a chilling tale of horror and strife across the sinister bounty and scalping business of the western frontier.",
```
Comment (ToxiClay) [inline data/json/items/book/misc.json:354]:
```suggestion
    "description": "Award-winning author Cornac McCaffordy weaves a chilling tale of horror and strife, following the compellingly lucrative, seductively dangerous bounty hunting and scalping trade across the western frontier.",
```

**Comment by ToxiClay** [comment]
Confirming proper function and spawn of added book.
image

**Comment by kevingranade** [comment]
Needs some spawn locations, otherwise no problem having it.

**Comment by ToxiClay** [comment]
@SpasmodicReveler -- look in `data\json\itemgroups\SUS\library.json` for places where you can define where the book pops up. The groups in this file are referenced all over, so adding the item here will populate it throughout the game world.

**Comment by I-am-Erk** [comment]
Does this need to be an entire separate book? It seems better suited to being a snippet title of an existing book...

**Comment by Maleclypse** [comment]
Once this is merged I'll consolidate all the western books into a single western item with snippets

**Pull request merged**